docs/source/customization.mdx

```diff
@@ -32,17 +32,6 @@ All of the trainers in TRL can be run on multiple GPUs together with DeepSpeed Z
 accelerate launch --config_file=examples/accelerate_configs/deepspeed_zero{1,2,3}.yaml --num_processes {NUM_GPUS} path_to_your_script.py --all_arguments_of_the_script
 ```
 
-Note that for ZeRO-3, a small tweak is needed to initialize your reward model on the correct device via the `zero3_init_context_manager` context manager. Here is an example adapted for the [`sentiment_tuning`](https://github.com/huggingface/trl/blob/main/examples/scripts/sentiment_tuning.py):
```

Comment:
As discussed offline, this is no longer needed for `deepspeed>=0.9.5`

Reply:
Indeed, ZeRO-3 hangs after 10 steps without the context manager. Curiously, this happens exactly after the `pipleine()` method throws a warning that one is using the pipeline sequentially and presumably this is the root cause